### DISCUSSION TITLE
请问下 淘宝小程序 截屏怎么实现

### DISCUSSION THREAD

**Discussion by SongBana** [layabox/LayaAir#1358] [General]
var htmlCanvas = Laya.stage.drawToCanvas(Laya.stage.width, Laya.stage.height, 0, 0);
        var canvas = htmlCanvas.toBase64("image/png", 0.5);
这个方法在淘宝ide会报错

**Comment by LayaCharley** [comment]
请查看LayaAir3引擎的示例，示例中提供了更多的新的截屏方案。https://layaair.com/3.x/demo/
